app/src/main/java/net/gsantner/markor/format/general/CommonTextActions.java

```diff
@@ -55,9 +61,16 @@
     private final Activity _activity;
     private final HighlightingEditor _hlEditor;
 
+    protected AppSettings _appSettings;
+    private int _tabWidth;
+    protected Context _context;
```

Comment:
We dont need to preserve context as member right? Please remove as it may leak memory/data